.gitignore

```diff
@@ -33,3 +33,6 @@ profile.cov
 
 # IdeaIDE
 .idea
+
+# dashboard
+/dashboard/assets/node_modules
```

Comment:
Please fix all your files so they have a newline at the end.

Reply:
Also skip the bundle.js, since it's an intermediate build step between source jsx and the final assets.go